pyiron_contrib/protocol/compound/nudged_elastic_band.py

```diff
@@ -195,23 +202,56 @@ def plot_elastic_band(self, ax=None, frame=None, plot_kwargs=None):
         ax.xaxis.set_major_locator(MaxNLocator(integer=True))
         return ax
 
-    def get_forward_barrier(self, frame=None):
+    def _get_directional_barrier(self, frame=None, anchor_element=0, use_minima=False):
+        energies = self._get_energies(frame=frame)
+        if use_minima:
+            reference = energies.min()
+        else:
+            reference = energies[anchor_element]
+        return energies.max() - reference
+
+    def get_forward_barrier(self, frame=None, use_minima=False):
         """
         Get the energy barrier from the 0th image to the highest energy (saddle state).
+
+        Args:
+            frame (int): A particular dump. (Default is None, the final dump.)
+            use_minima (bool): Whether to use the minima of the energies to compute tha barrier. (Default is
+                False, use the 0th value.)
+
+        Returns:
+            (float): the forward migration barrier.
         """
-        energies = self._get_energies(frame=frame)
-        return np.amax(energies) - energies[0]
+        return self._get_directional_barrier(frame=frame, use_minima=use_minima)
 
-    def get_reverse_barrier(self, frame=None):
+    def get_reverse_barrier(self, frame=None, use_minima=False):
         """
         Get the energy barrier from the final image to the highest energy (saddle state).
+
+        Args:
+            frame (int): A particular dump. (Default is None, the final dump.)
+            use_minima (bool): Whether to use the minima of the energies to compute tha barrier. (Default is
+                False, use the nth value.)
+
+        Returns:
+            (float): the backward migration barrier.
         """
-        energies = self._get_energies(frame=frame)
-        return np.amax(energies) - energies[-1]
+        return self._get_directional_barrier(frame=frame, anchor_element=-1, use_minima=use_minima)
 
-    def get_barrier(self, frame=None):
-        return self.get_forward_barrier(frame=frame)
+    def get_barrier(self, frame=None, use_minima=True):
+        """
```

Comment:
You can get rid of this docstring and just say `get_barrier.__doc__ = get_forward_barrier.__doc__` right below the function definition to avoid repeated lines. (Or maybe `NEB.get_forward_barrier.__doc__` or whatever the class name is.)